pkgs/test/test/util/fake_test.dart

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+@TestOn('vm')
```

Comment:
It probably makes sense to remove this actually, this class should be usable on all platforms